### ISSUE TITLE
Destination Gate/Parking Selection Functionality

### ISSUE THREAD

**Issue by betterpilot** [albar965/littlenavmap#582]
After landing, ATC often assigns a gate that the pilot may not be able to easily locate on the moving map due to limited concourse information showing on the map overlay, or pilot being generally unfamiliar with the airport. 

It would be extremely beneficial to allow the user to select a gate using the same functionality as the "Select Start Position For Flight Plan" that highlights the gate/parking spot for the pilot after landing at the destination once their gate is assigned by ATC. 

I am proposing a feature that clones the functionality of the "Select Start Position For Flight Plan" feature but repurposed for the destination airport which would highlight the gate for better situational awareness for the taxi-in phase of flight when landing at destination airports.

**Comment by albar965** [comment]
Makes sense and is not too much effort. Assigning to 2.8 which is the next big release after 2.6.

**Comment by wolfallein** [comment]
Is it possible to revisit this enhancement?

**Comment by albar965** [comment]
Well ... yes. That's why it has the `reschedule` tag. But it will take a while.

**Comment by wolfallein** [comment]
Sorry, I didn't see the `reschedule` tag 😄 

Thank you for the amazing work!
